specification/offazurespringboot/resource-manager/Microsoft.OffAzureSpringBoot/preview/2023-01-01-preview/examples/springbootapps_CreateOrUpdate_MaximumSet_Gen.json

```diff
@@ -0,0 +1,214 @@
+{
+  "parameters": {
+    "siteName": "pdfosfhtemfsaglvwjdyqlyeipucrd",
```

Comment:
Would you update the examples to use more meaningful values? Please also update other places.

Reply:
I used the ova tool to generate the examples automatically.